src/Request/Data/Invoice/InvoiceCreateData.php

```diff
@@ -9,12 +9,32 @@
 namespace ZEROSPAM\Freshbooks\Request\Data\Invoice;
 
 use ZEROSPAM\Freshbooks\Request\Data\ArrayableData;
+use ZEROSPAM\Freshbooks\Request\Data\Client\LatePayment\CommonWriteLatePaymentReminders;
 use ZEROSPAM\Freshbooks\Request\Data\Common\CommonWritableFields;
 use ZEROSPAM\Freshbooks\Request\Data\Common\CommonWritableOnCreateFields;
+use ZEROSPAM\Freshbooks\Response\Invoice\Presentation\PresentationResponse;
 
 class InvoiceCreateData extends ArrayableData
 {
+
+    /**
+     * @var PresentationResponse
+     */
+    private $presentation;
+
+
     use CommonWritableFields,
         CommonWritableOnCreateFields,
-        WritableInvoiceFieldsTrait;
+        WritableInvoiceFieldsTrait,
+        CommonWriteLatePaymentReminders;
+
+    /**
+     * Set the presentation of the invoice
```

Comment:
This is voluntary as the data NEEDS to come from the presentation "api" since it needs the "invoiceid" and other weirdness.